observation_portal/observations/models.py

```diff
@@ -1,5 +1,4 @@
 from django.db import models
-from django.contrib.postgres.fields import JSONField
```

Comment:
Not sure about all of these changes - I am guessing they're from the dependency upgrades? I'll let Jon comment

Reply:
The JSONField import removal is because its now included in the base django 3 models.